src/app.service.ts

```diff
@@ -3,6 +3,12 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return 'Hello World!';
+    return 'Hello World how is it looking right now!';
   }
-}
+
+  get(): string {
+    console.log('hello world, how is ig!')
+    const maths = 1 +2+5;
+    return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
```

Comment:
PullProofAI

The `get()` method in `AppService` contains unnecessary console logs and a calculation that seems out of place. It's recommended to clean up the method for better readability and maintainability.

```diff
-     console.log('hello world, how is ig!')
-     const maths = 1 +2+5;
-     return `${maths} Hello Prima. how are youdoing what is the total of given number and 20%`;
+     return 'Hello Prima. How are you doing?';
```


<!-- This is an auto-generated comment by PullProofAI -->